src/test/java/kitchenpos/domain/menu/MenuProductTest.java

```diff
@@ -6,71 +6,105 @@
 import java.math.BigDecimal;
 import java.util.Arrays;
 
+import javax.persistence.EntityNotFoundException;
+
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.FilterType;
+import org.springframework.stereotype.Component;
 
 import kitchenpos.application.fixture.MenuFixtureFactory;
 import kitchenpos.application.fixture.MenuGroupFixtureFactory;
+import kitchenpos.application.fixture.MenuProductFixtureFactory;
 import kitchenpos.application.fixture.ProductFixtureFactory;
+import kitchenpos.application.menu.MenuValidator;
 import kitchenpos.domain.Quantity;
 import kitchenpos.domain.menugroup.MenuGroup;
+import kitchenpos.domain.menugroup.MenuGroupRepository;
 import kitchenpos.domain.product.Product;
+import kitchenpos.domain.product.ProductRepository;
 
+@DataJpaTest(includeFilters = @ComponentScan.Filter(type = FilterType.ANNOTATION, classes = Component.class))
```

Comment:
Q2. Validator 를 @component Bean 으로 만들면서, 기존 ServiceTest 를 Mock 테스트에서 DataJpaTest 로 변경하였습니다. 아무래도 DataJpaTest 를 사용하니 성능면에서 다소 느린것을 체감할 수 있었는데 이부분을 개선할 수 있는 방법이 있을까요?

해당 부분에 대한 성능을 개선해보려면 필터를 추가하는 것보다 필요한 빈을 Import로 추가하는 방법도 있을 것 같아요 :)

아래 링크의 내용이 같은 상황은 아니지만 DataJpaTest에 대해 생각해볼 수 있는 포인트가 될 것 같아서 남겨놓을게요 😄 

https://www.inflearn.com/questions/42585

```suggestion
@DataJpaTest
@Import(value = MenuValidator.class)
```